platform/rct/hal/usart_stm32/usart_device_stm32.c

```diff
@@ -47,16 +48,25 @@
 #define DEFAULT_TELEMETRY_BAUD_RATE	115200
 #define DEFAULT_GPS_BAUD_RATE		9600
 
+/* 32 bytes is good up to a baud of 230400 (28 chars / ms) */
+#define WIFI_DMA_RX_BUFF_SIZE	32
+
```

Comment:
You might want to pad this just a bit more.  Since they're soft timers, you don't get a deadline guarantee.